bpm_summary_ds/bpm_summary.py

```diff
@@ -0,0 +1,107 @@
+"""
+This module contains device class BPMSummary and run method for it.
+"""
+
+# Imports
+from tango import DevState, AttrWriteType
+from facadedevice import Facade, combined_attribute
+from numpy import std, mean
+from math import sqrt
+
+
+class BPMSummary(Facade):
+    """
+    This Tango Device Class implements Facade Device to summarize Beam
+    Position Monitors. It allows to compute following values:
+
+    * mean
+    * root mean square
+    * maximum peak to peak value
+    * standard deviation
+    * maximum deviation
+
+    Those calculations are performed on BPMs' attributes provided via BPMList.
+    They can be provided in two ways:
+
+    * explicit list of all attributes for calculations.
+
+    It's useful when BPMs are on different Device Servers. For example:
+    some/bpm/somewhere/attr, some/other/bpm/attr, here/some/other/attr
+
+    * simple comprehension containing '*' character in place which should accept any value.
+
+    For example: some/bpm/*/attr. This will allow BPMSummary Device to use in
+    calculations every single device which name fulfill this comprehension, like:
+
+    -- some/bpm/here/attr
+
+    -- some/bpm/there/attr
+
+    -- some/bpm/out_there/attr
+
+    -- etc.
+
+    """
```

Comment:
One more thing: to have a nice looking rendered documentation, please surround all example device names with backticks so that they're shown as code in the docs. Please also do the same for `BPMList`.